spec/features/user_logs_in_spec.rb

```diff
@@ -4,13 +4,13 @@
   scenario "logs in" do
     user = FactoryGirl.create(:user)
 
-    visit login_path
+    visit sign_in_path
```

Comment:
Can you use `rails generate clearance:specs` to generate clearance's feature specs and use those?